addons/building_markers/script_component.hpp

```diff
@@ -0,0 +1,17 @@
+#define COMPONENT building_markers
+#define COMPONENT_BEAUTIFIED Building_Markers
```

Comment:
No underscore in pretty name.

Reply:
[fe8a983](https://github.com/zen-mod/ZEN/pull/704/commits/fe8a983ca4499cb423821126db5ee8e529416f11)